packages/jira-adapter/src/client/client.ts

```diff
@@ -38,6 +40,18 @@ const DEFAULT_PAGE_SIZE: Required<clientUtils.ClientPageSizeConfig> = {
   get: 1000,
 }
 
+export type graphQLResponseType = {
+  data: unknown
+  errors?: unknown
+}
+
+const GRAPHQL_RESPONSE_SCHEME = Joi.object({
+  data: Joi.object().required(),
+  errors: Joi.object().optional(),
```

Comment:
since it is not necessarily an object
```suggestion
  data: Joi.required(),
  errors: Joi.optional(),
```